sdk/ml/azure-ai-ml/tests/pipeline_job/e2etests/test_pipeline_job.py

```diff
@@ -1696,6 +1696,22 @@ def check_name_version_and_register_succeed(output, asset_name):
         assert pipeline_job.jobs["compare"].outputs.best_model.name == "best_model"
         assert pipeline_job.jobs["compare"].outputs.best_model.version == random_version
 
+    @pytest.mark.skipif(condition=not is_live(), reason="component version changes across tests")
```

Comment:
Add a task number to track this, can directly use the existing task. 2177353 and update these case to it

Reply:
Updated the skip reason with this work item.